CSS/style.css

```diff
@@ -7,15 +7,19 @@
 }
 
 .app {
-    
     height: 100vw;
+    background-image: url(../Images/background.png);
+    background-repeat: no-repeat;
+    background-size: contain;
```

Comment:
You can use `background-size: cover;` to make the image fully cover the background. Also might add `background-position: center` to center the image position if needed.

Reply:
When I change it to background-size: cover; it does not look as same as the given design on figma.